content/docs/k8s/gateway-api.mdx

```diff
@@ -2,15 +2,17 @@
 title: Gateway API
 sidebar_position: 3
 lang: en-US
-keywords: [kubernetes, k8s, gateway api]
+keywords: [kubernetes, k8s, gateway api, ingress, ingress controller]
```

Comment:
Added this to https://github.com/pomerium/documentation/pull/1689.

(Not really sure if these really matter at all or not.)